.github/workflows/repo-sync.yml

```diff
@@ -110,29 +110,6 @@ jobs:
               throw err
             }
 
-            console.log('Locking conversations to prevent spam')
-            try {
-              await github.rest.issues.lock({
-                ...context.repo,
-                issue_number: pull_number,
-                lock_reason: 'spam'
-              })
-              console.log('Locked the pull request to prevent spam')
-            } catch (error) {
-              console.error('Failed to lock the pull request.', error)
-              // Don't fail the workflow
-            }
-
-            console.log('Counting files changed')
```

Comment:
@heatlikeheatwave wouldn't we want to keep this part in? As we likely don't want to me making PRs if no files have changed.

Reply:
Added those back in. That was a mistake on my part